src/main/docs/guide/views/templates/react.adoc

```diff
@@ -0,0 +1,24 @@
+React server-side rendering (SSR) allows you to pre-render React components to HTML before the page is sent to the user.
+This improves performance by ensuring the page appears before any Javascript has loaded (albeit in a non-responsive
+state) and makes it easier for search engines to index your pages.
+
+Micronaut's support for React SSR has the following useful features:
```

Comment:
add a `{note}` that this module is experimental and subject to change

Reply:
I couldn't find an example of the syntax you mean, so wrote a `NOTE: This module is experimental` disclaimer.